examples/foundry/object_detection.ipynb

```diff
@@ -0,0 +1,376 @@
+      "source": [
+        "project = client.create_project(name=\"Foundry Image Demo\",\n",
+        "                                media_type=lb.MediaType.Image)\n",
+        "\n",
+        "project.setup_editor(ontology)"
+      ],
+      "cell_type": "code",
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "## Step 4: Create foundry application in UI\n",
+        "\n",
+        "Currently we do not support this workflow through the SDK\n",
+        "#### Workflow:\n",
+        "\n",
+        "1. Navigate to model and select ***Create*** > ***App***\n",
+        "\n",
+        "2. Select ***Amazon Rekognition*** and name your foundry application\n",
+        "\n",
+        "3. Customize your perimeters and then select ***Save & Create***"
+      ],
+      "cell_type": "markdown"
+    },
+    {
+      "metadata": {},
+      "source": [
+        "#Select your foundry application inside the UI and copy the APP ID from the top right corner\n",
+        "AMAZON_REKOGNITION_APP_ID = \"\""
+      ],
+      "cell_type": "code",
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "## Step 5: Run foundry app on data rows\n",
+        "\n",
+        "This step is meant to generate annotations that can later be reused as pre-labels in a project. You must provide your app ID from the previous step for this method to run, please see the [Foundry Apps Guide](https://docs.labelbox.com/docs/foundry-apps#run-app-using-sdk) for more information.\n"
+      ],
+      "cell_type": "markdown"
+    },
+    {
+      "metadata": {},
+      "source": [
+        "task = client.run_foundry_app(model_run_name=f\"Amazon-{str(uuid.uuid4())}\",\n",
+        "                              data_rows=lb.GlobalKeys(\n",
+        "                                  [global_key] # Provide a list of global keys \n",
+        "                                  ), \n",
+        "                              app_id=AMAZON_REKOGNITION_APP_ID)\n",
+        "\n",
+        "task.wait_till_done()\n",
+        "\n",
+        "print(f\"Errors: {task.errors}\") \n",
+        "\n",
+        "#Obtain model run ID from task\n",
+        "MODEL_RUN_ID = task.metadata[\"modelRunId\"]"
+      ],
+      "cell_type": "code",
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Errors: None\n"
+          ]
+        }
+      ],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "## Step 6: Map ontology through the UI\n",
+        "\n",
+        "Something like: Mapping a model's ontology to a project's ontology is currently not supported through the SDK, however, to showcase how to send foundry predictions to a project, we are going to generate the mapping of the foundry app ontology to the project ontology through the UI.\n",
```

Comment:
Could you remove "Something like:" -  Once removed this all looks good and you can merge!